docs/quickstarts/packet.md

```diff
@@ -27,17 +27,19 @@ The guide uses [Amazon Route 53](https://aws.amazon.com/route53/) as a DNS provi
 
 ## Requirements
 
-* Basic understanding of Kubernetes concepts.
-* Packet account, Project ID and auth token (sometimes also referred to as [User Level API
-  key](https://www.packet.com/developers/docs/API/getting-started/)).
-* AWS account and IAM credentials (optional for Route53 DNS configuration).
-* AWS Route53 DNS Zone (registered Domain Name or delegated subdomain).
-* Terraform v0.12.x and [terraform-provider-ct](https://github.com/poseidon/terraform-provider-ct) v0.5.0
-  installed locally.
-* Local BGP enabled. More information on how to enable Local BGP for the Packet Project is found in
-  the [Packet support document](https://support.packet.com/kb/articles/bgp).
-* An SSH key pair for management access.
-* `kubectl` installed locally to access the Kubernetes cluster.
+* A Packet account with
+  [local BGP](https://www.packet.com/developers/docs/network/advanced/local-and-global-bgp/)
+  enabled.
```

Comment:
@rata I don't understand what change you are proposing. Could you be more concrete?

Reply:
To be honest, it was > 1 month ago, I don't remember. I think I was just asking, not pointing a particular currection. My question, IIRC, is that given that MetalLB is not deployed here, don't we want to disable BGP?

Either never enable it in the project (so remove this line) and disable it in the workers (not sure if the controllers will work, though), or enable it in the project (so, keep this line) but disable it in the workers (so, add the knob there).

My intention was to ask if this is intentional and if you thought of disabling BGP given that MetalLB is not deployed. IIUC, the tutorial about MetalLB can show how to enable it (in the project and/or workers, it should be just changing the config and `cluster apply`).

What do you think?

Reply:
So IIUC the summary is "are you sure we want to say BGP should be enabled?". Thanks :+1:
I guess we don't need it since we don't deploy LB + ingress within this guide. I'll remove it, but I'm going to first verify it is possible to enable BGP retroactively since the very next step after completing this guide will likely be enabling LB + ingress.

Reply:
Makes sense! If you prefer to not overload the work on this PR, keep it and open an issue to check it later. Whatever you think is better/easier :)

Reply:
Having it enabled here makes total sense, then. Sorry for the noise! :)